tests/test_generic/tests/test_xml.py

```diff
@@ -46,18 +53,35 @@ def _check_files_in_path(self, module):
                     else:
                         manifest_content = content
                     continue
+
                 self._check_xml_style(content, module, file_name)
                 self._check_update_status(content, file_name)
                 self._check_useless_models(content, file_name)
                 self._check_useless_fields_on_models(content, file_name)
                 self._check_knowledge_article_is_published(content, file_name)
                 self._check_duplicate_records(content, file_name)
                 self._check_website_published_false(module, file_name)
+                self._check_static_files_usage_in_xml(content, in_use_files)
                 if root.split('/')[-1] == 'data':
                     self._check_is_published_false(module, file_name)
                     if not is_studio_required:
                         is_studio_required = self._check_studio(content, file_name)
         self._check_manifest(manifest_content, is_studio_required)
+        in_use_files = set(file.lstrip('/') for file in in_use_files)
```

Comment:
Don't you need to remove the name of the module as well?

Reply:
The name of the module is in the static files list.